triton-server/src/notification.ts

```diff
@@ -183,14 +183,14 @@ export const sendTestEmail = (body: string) => {
     const transporter = nodemailer.createTransport({
         service: "gmail", // other mailer can be used but right now default is gmail
         auth: {
-            user: "sebastianamouzegar@gmail.com",
-            pass: "tlba fptj scli xfdy",
+            user: "", // your email(should be a gmail if using gmail email transporter service)
+            pass: "", // your gmail secret passkey to allow the app to use the gmail api through your gmail
         },
     })
 
     const mailOptions = {
-        from: "sebastianamouzegar@gmail.com",
-        to: "sebastian.amouzegar@computationalgenomics.ca",
+        from: "",
+        to: "",
         subject: "Sending Email using Node.js",
         text: body,
     }
```

Comment:
not if you want to use the test emails. 
if you want to use the test emails you have to provide 4 important variables
`    const transporter = nodemailer.createTransport({
        service: "gmail", // other mailer can be used but right now default is gmail
        auth: {
            user: "", // your email(should be a gmail if using gmail email transporter service)
            pass: "", // your gmail secret passkey to allow the app to use the gmail api through your gmail
        },
    })

    const mailOptions = {
        from: "",
        to: "",
        subject: "Sending Email using Node.js",
        text: body,
    }
i will add more information on the file just in case there is confusion`